pkg/controller/chaosscheduler/updateStatus.go

```diff
@@ -48,22 +54,6 @@ func (r *ReconcileChaosScheduler) updateActiveStatus(cs *chaosTypes.SchedulerInf
 	return nil
 }
 
-// deleteJob reaps a job, deleting the job, the pods and the reference in the active list
-func (r *ReconcileChaosScheduler) deleteEngine(cs *chaosTypes.SchedulerInfo, engine *operatorV1.ChaosEngine) bool {
```

Comment:
Are we not using this function anywhere?? I guess we are using it somewhere

Reply:
yes we are not using it

Reply:
This function removes an active engine This function deletes an engine from the active list when it is completed.

Reply:
we are using different function to remove the engine from the active list. This function just delete the chaos engine but we are not using it anywhere i have verified